docs/release-notes/2.35.0.md

```diff
@@ -0,0 +1,139 @@
+
+#### Cluster membership check
+
+To avoid execution errors, PMM no longer allows scheduling backups on MongoDB Services that are not managed as a cluster.
+
+You can check which of your current Services are not members of a cluster on the **PMM Inventory > Services** page. Services that do not specify a cluster name in the **Other Details** column should be removed and re-added using the following command:
+
+<pre>pmm-admin add mongodb
+username=pmm_mongodb --password=password 
+query-source=profiler <mark>--cluster=mycluster</mark></code></pre>
+
+
+### Backup restore logs accessible from PMM
+
+With this release, we are adding the option to check the logs for restores from a MongoDB database backup.
+This complements the existing option to check the logs for the backup process and makes it easier to gather data for compliance, auditing, and troubleshooting purposes.
+
+Instead of having to look through PBM log files, you can now access the **Log** option in PMM, which is now automatically displayed next to each MongoDB backup restore entry on the **Backup > Restores** page.
+
+### Components Upgrade
+
+MongoDB exporter has been upgraded to 0.37.0.
+
+## New Features
+
+- [PMM-10325](https://jira.percona.com/browse/PMM-10325) - Starting with PMM 2.35.0, for enhanced security, you can change the admin password even after `pmm-agent` is registered on the PMM Server.
+
+- [PMM-10550](https://jira.percona.com/browse/PMM-10550) - [Backup Management]: PMM now automatically displays a `Logs` option next to each MongoDB backup restore entry on the **Backup > Restores** page.
+
+- [11302](https://jira.percona.com/browse/PMM-11302) - [DBaaS]: Starting with PMM 2.35.0, you can add a backup schedule while creating DB clusters in DBaaS.
+
+- [PMM-10962](https://jira.percona.com/browse/PMM-10962) - Starting with PMM 2.35.0, we have introduced Access Control. Roles are a vital part of Access Control that provide the users with access to specified metrics. You can create, edit, and delete user roles. The roles can be assigned to the users with the **Configuration > Users** tab. For more information, see [documentation]().
+
+## Improvements
+
+- [PMM-10471](https://jira.percona.com/browse/PMM-10471) - [Backup Management]: PMM now automatically checks the current MongoDB node to ensure that the connected PMM Agents have the tools to execute and finalize the backup.
+
+- [PMM-11113](https://jira.percona.com/browse/PMM-11113) - Enhanced user experience for the **Percona Platform** page under **Configuration > Settings**. The new version of this page highlights the advantages of connecting your PMM instance to Percona Platform and makes it easier to generate and use a connection token.  
+
+- [PMM-11499](https://jira.percona.com/browse/PMM-11499) - You can now enable Access Control from the **Configuration > Settings > Advaned setting** page on the PMM UI.
```

Comment:
what do we put this to improvement part? Isnt it the part of Access Control? We're not improving it, we added it on this release. @matejkubinec what do you think?

Reply:
I agree, it's not worth mentioning it under improvements.

Reply:
Deleted it.